lib/ansible/config/base.yml

```
@@ -1367,6 +1367,15 @@ GALAXY_COLLECTION_SKELETON_IGNORE:
   ini:
   - {key: collection_skeleton_ignore, section: galaxy}
   type: list
+GALAXY_COLLECTIONS_PATH_WARNINGS:
```

`version_added` should be 2.16 at this point.

```suggestion
GALAXY_COLLECTIONS_PATH_WARNING:
```